keypom/keypom-airfoil
# Refactor claim pages and utils

# Description
- Refactored Token and NFT claims into a generic component
- Refactored Tickets flow, all ticket pages are governed by a single context that will route according to key use
- Removed all unused utils

Fixes #195 #185 

# How to test
Show steps to replicate and test the feature/fixes in this PR

1. check out the preview link
2. claim token, nft
3. go through ticket flow
4. ensure everything works as usual

# Screenshots/Screen Recording of Implementation
